### ISSUE TITLE
Axial spondyloarthritis

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6541]
**Preferred term label:** Axial spondyloarthritis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Axial spondyloarthritis is an umbrella term for different types of inflammatory arthritis that affect predominantly the spine and to a lesser degree the other joints.

**Parent term (use hpo.jax.org/app)**
Arthritis HP:0001369

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
spondylo =vertebral body and so "axial" does not add addditional information. "Axial spondyloarthritis " defined as above is a disease, but this would not be a good HPO term. The HPO would have a term such as "Vertebral body arthritis" or perhaps spondyloarthritis defined to be specifically the phenotypic manifestation.

**Comment by eswietlik** [comment]
It looks like spondyloarthritis was divided into two subcategories (axial and peripheral) in 2009 PMID
19297344; PMID 21109520 (study done at Charite by Rudwaleit M). 
My patients have the following recorded (among terms relevant to this topic) in the clinical notes
Patient 1) ulcerative colitis, axial spondyloarthritis, B27 negative, joint stiffness,  IPAH
Patient 2) ankylosing spondylitis (axial spondyloarthritis), B27 positive, IPAH, uveitis, lower back pain, MRI signs of sacroiliitis 
Patient 3) psoriasis, axial spondyloarthritis, morning stiffness, B27 negative, IPAH
Patient 4) axial spondylarthritis, RF negative, knee osteoarthritis, IPAH, small PFO, bibasal mild lung fibrosis, back pain

So according to the paper and Patient 2, this is rather a disease or disease subtype. In other cases, axial spondyloarthritis is used more to describe the abnormality.

**Comment by pnrobinson** [comment]
@eswietlik  I think the phenotypic feature is best described as ' spondylitis'  or vertebral inflammation

**Comment by eswietlik** [comment]
Thank you. Let's go with spondylitis. Emilia

niedz., 31 sty 2021 o 18:45 Peter Robinson <notifications@github.com>
napisał(a):

**Comment by pnrobinson** [comment]
added
